ImageInLib/ImageInLib/src/common_functions.c

```diff
@@ -159,48 +159,105 @@ size_t x_flat(const size_t rowIndex, const size_t columnIndex, const size_t heig
 	return rowIndex + rowLength * (columnIndex + columnLength * heightIndex); // x + xlen * (y + ylen * z)
 }
 //==============================================================================
-void rescaleNewRange(dataType ** dta, size_t height, size_t length, size_t width, dataType min_new, dataType max_new, dataType max_dta, dataType min_dta)
```

Comment:
I think we don't need to give the max and min as parameters, because If yes, that means we will need to seek the min and the max before inside the function. But how I modified, we just need the new min and max. I think I took the idea in the file segmentation.c, the same function is there and the old min and max re found inside the function